examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -252,6 +252,12 @@ impl OakAbiTestService for FrontendNode {
             "NodeCreateRaw",
             (Self::test_node_create_raw, Count::Unchanged),
         );
+        tests.insert("ChannelLabel", (Self::test_channel_label, Count::Unchanged));
+        tests.insert("NodeLabel", (Self::test_node_label, Count::Unchanged));
+        tests.insert(
+            "NodePrivilege",
+            (Self::test_node_privilege, Count::Unchanged),
+        );
```

Comment:
Maybe add some `_raw` variants that use the `oak_abi` entrypoints directly and exercise invalid behaviour (wrong sizes, bad pointers) ?